hicommonwealth/commonwealth
# Contest - rate limiting UI

## Link to Issue
Closes: #8474
Closes: #8473

## Description of Changes
- added new banner in thread creation page 
- added errors in thread creation page
- modified initial state for toggles in launch contest view

## Test Plan
- go to launch contest form
- `general` and `all` should be toggled off by default
- create contest with some topic 
- go to thread creation page
- pick the topic that was selected fot contest
- you should see banner informing about the contest running for this topic
- try to create 2 thread in this topic
- if you will try to create third one, there should be an error in the UI + create thread button disabled
- also if you want to create thread in the contest and you dont have at least 0.005 eth, you would also see the error and the button will be disabled

See the video


https://github.com/user-attachments/assets/53e80e10-a764-4e31-84e0-5c6145c38ff0




## Deployment Plan

1. n/a

## Other Considerations

- this PR should be merged in first https://github.com/hicommonwealth/commonwealth/pull/8475
